packages/client/__tests__/client.test.js

```diff
@@ -37,5 +57,118 @@ describe('API - Client', () => {
 
       expect(client.http.version).toBe(2)
     })
+
+    it('should throw an Error if the API version is falsy', async () => {
+      try {
+        expect(async () => client.setVersion(0)).toThrow()
+        expect().fail('Should fail on the previous line')
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error)
+      }
+    })
+  })
+
+  describe('static findPeers', () => {
+    it('should throw an Error if the network does not have initial peers', async () => {
+      try {
+        expect(async () => Client.findPeers('wrong')).toThrow()
+        expect().fail('Should fail on the previous line')
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error)
+      }
+    })
+
+    xit('should connect randomly to the initial peers', () => {
+    })
+
+    it('should return a sorted list of peers', async () => {
+      const data = {
+        success: true,
+        peers
+      }
+      httpMock.onGet('peers').reply(200, { data })
+
+      const foundPeers = await Client.findPeers('mainnet')
+      expect(foundPeers).toEqual([
+        peers[1],
+        peers[3],
+        peers[0],
+        peers[2]
+      ])
+    })
+
+    it('should ignore local peers', async () => {
+      ;['127.0.0.1', '::1'].forEach(async ip => {
```

Comment:
semi colon at the start of the line

Reply:
may also be worth adding `::ffff:127.0.0.1` which is in the whitelists

Reply:
I'm not sure about it. It doesn't look like a valid IP address